domains/gxlpes.json

```diff
@@ -5,7 +5,8 @@
         "email": "guilhermxlopes@gmail.com"
       },
       "record": {
-        "URL": "https://gxlpes.github.io/portfolio-website/"
+        "URL": "https://gxlpes.github.io/portfolio-website/",
```

Comment:
You cannot have URL and CNAME record together.
You likely want to use a CNAME record, read more about it [here](https://www.is-a.dev/docs/github_pages).

Reply:
Ok, sorry about that.

Reply:
> Ok, sorry about that.
> 
> Do I have to approve your suggestion or something like that?

You have to update the file and fix the record structure, otherwise it cannot be merged. Make sure that the checks are passing.